packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -816,6 +822,19 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
         return fullTooltip;
     }
 
+    private generateTooltipTable(tooltip: { [key: string]: string }) {
+        return (
+            <table>
+                {Object.keys(tooltip).map(key => (
+                    <tr key={key}>
+                        <td style={{ textAlign: 'left' }}> {key} </td>
+                        <td style={{ textAlign: 'left' }}> {tooltip[key]} </td>
```

Comment:
Seems odd formatting to add spaces around the {...} ? Does it get stripped in the html?

Reply:
I mean around {key} and {tooltip[key]} inside the <td></td> html tag.